packages/instant-apps-components/src/components/instant-apps-app-guide/AppGuide/AppGuideViewModel.ts

```diff
@@ -2,6 +2,7 @@ import { AppGuidePage } from "./interfaces/interfaces";
 
 class AppGuideViewModel {
   pages: AppGuidePage[];
+  headerTitle: string;
```

Comment:
What do you think of naming this to `headerText` since that's what's used in the t9n files?

Reply:
Yep, makes sense; I'll update the private method to `_updateHeaderText` as well.